src/uct/ib/base/ib_iface.c

```diff
@@ -529,7 +533,15 @@ void uct_ib_iface_fill_ah_attr_from_gid_lid(uct_ib_iface_t *iface, uint16_t lid,
     ah_attr->grh.traffic_class = iface->config.traffic_class;
 
     if (uct_ib_iface_is_roce(iface)) {
-        udp_sport               = iface->config.roce_path_factor * path_index;
+        udp_sport              = iface->config.roce_path_factor * path_index;
```

Comment:
1. fix indent
2. put this line as "else" of the "if" in line 537, and update the calculation in lines 542-543 accordingly

Reply:
@yosefe Fixed